g2p_registry_base/models/reg_id.py

```diff
@@ -16,6 +16,8 @@ class G2PRegistrantID(models.Model):
     )
     id_type = fields.Many2one("g2p.id.type", "ID Type", required=True)
     value = fields.Char(size=100)
+    status = fields.Char()
```

Comment:
What is the expected values for this?

Reply:
The `status` and `error` (optional) fields came from an ID System like MOSIP in mind. If there were an external entity feeding data/IDs into OpenG2P and if there was some error during that, these fields could help the Registrar/Program Manager visualize that.

Reply:
Then it is better to be in a separate module that adds it when it is needed. 
Also I would not call it status but sync status or something like this.